package.json

```
@@ -6,7 +6,9 @@
   "type": "module",
   "files": [
     "app/javascript/**/*.js",
-    "plugin.js"
+    "app/views/**/*.{arb,erb,html,rb}",
```

We don't need to specify all the file extensions, just these two as `npm publish --dry-run` confirms they will all be included. I'll update the Tailwind config template separately.

```suggestion
    "app/views/**/*.{arb,erb}",
```

---

Thanks for the feedback, I've updated based on your suggestion.